modules/monitoring-adding-a-secret-to-the-alertmanager-configuration.adoc

```diff
@@ -138,3 +138,50 @@ endif::openshift-dedicated,openshift-rosa[]
 
 . Save the file to apply the changes to the `ConfigMap` object. The new configuration is applied automatically.
 
+.Examples
+
+These examples show settings that configure Alertmanager in the `openshift-monitoring` namespace to authenticate with a webhook endpoint by using an API token that is configured in a `Secret` object.
+
+The following sample YAML shows a `Secret` object with authentication credentials to be referenced in the Alertmanager config map settings for the Cluster Monitoring Operator (CMO):
+
+[source,yaml]
+----
+apiVersion: v1
+data: 
+  api_token: m8VjfmV2Cg== <1>
+kind: Secret
```

Comment:
Isn't the "kind" usually shown after the apiVersion? It might still work, but it threw me off at first glance when I couldn't tell right away which object type this was

Reply:
@bergerhoffer My understanding is that the order isn't relevant, But if the practice is to put it in a certain place, I will change it to maintain consistency with other samples.